### PR TITLE
feat(index.js-book.js): change from client to pool for transaction, a…

### PR DESCRIPTION
**Pull request by yenting-biao**
…nd modify the code in book.js

應該是這樣嗎？不太確定所以我先 assign 給兩個人都看過

[https://node-postgres.com/features/transactions](https://node-postgres.com/features/transactions)

### CODE REVIEW

**Comment by quevenliu** [comment]
I think your code is correct. And maybe for transaction, we just need to use `db.query("BEGIN");` and do not need to use other special functions? 

After the PR is merged I will edit my code accordingly.

**Review by quevenliu — APPROVED** [review]
I think your code is correct. I think the PR can be merged if your test on your local machine is correct. 

And maybe for transaction, we just need to use `db.query("BEGIN");` and do not need to use other special functions?

After the PR is merged I will edit my code accordingly.

**Comment by yenting-biao** [comment]
I finished transaction part and also found some bugs to be fixed.

**Pull request merged**
